Runtime/Client/LootLockerEndPoints.cs

```diff
@@ -235,6 +235,9 @@ public class LootLockerEndPoints
         public static EndPointClass getAllMemberRanks = new EndPointClass("leaderboards/member/{0}?count={1}", LootLockerHTTPMethod.GET);
         public static EndPointClass getScoreList = new EndPointClass("leaderboards/{0}/list?count={1}", LootLockerHTTPMethod.GET);
         public static EndPointClass submitScore = new EndPointClass("leaderboards/{0}/submit", LootLockerHTTPMethod.POST);
+        public static EndPointClass getLeaderboardData = new EndPointClass("leaderboards/{0}/info", LootLockerHTTPMethod.GET);
```

Comment:
It seems you accidentally included the changes "Leaderboard Details" changes in this PR. I will not comment more on that than saying that you'll need to fix that :P